kubenow/gluster-s3/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Default values for a Gluster-S3 deployment.
+# this is a yaml-formatted file.
+# declare variables to be passed into your templates.
+releaseName: s3-interface
```

Comment:
release name can be passed trough helm, no need to have it here

Reply:
this is the default value

Reply:
good if the user wants to deploy fast